packages/office-ui-fabric-react/src/utilities/color/colors.test.ts

```diff
@@ -141,16 +141,14 @@ describe('color utilities', () => {
   });
 
   describe('cssColor', () => {
-    it('handles named colors', () => {
-      expect(cssColor('crimson')).toEqual({ r: 220, g: 20, b: 60, a: 100 });
-    });
+    // cssColor uses getComputedStyle() under the covers, which is incompletely implemented in headless browsers
+    // thus, we cannot fully test all cases here, such as for named colors
 
     it('handles invalid hex input', () => {
       expect(cssColor(undefined as any)).toBeUndefined();
       expect(cssColor(null as any)).toBeUndefined();
       expect(cssColor('')).toBeUndefined();
       expect(cssColor('000')).toBeUndefined(); // missing #
-      expect(cssColor('#0000')).toBeUndefined(); // wrong length
```

Comment:
this DOES fail as expected in the build, but that's because headless browsers don't support it, whereas a real browser does read it as hex-with-alpha